themes/bootstrap3/templates/feedback/response.phtml

```diff
@@ -12,7 +12,9 @@
   <<?=$headTag?>><?=$this->escapeHtml($title)?></<?=$headTag?>>
 <?php endif; ?>
 <?=$this->flashmessages()?>
-<div role="status" class="alert alert-success">
-  <?=$this->translate($form->getSubmitResponse()) ?>
-</div>
+<?php if (!empty($this->successMessage)): ?>
```

Comment:
We should always have a success message if we get this far, no? Otherwise the user could end up with an empty screen. On error should be directed back to the form.

Reply:
You are right, I've removed the `if`. Thanks.